sources/srd.py

```diff
@@ -170,1082 +170,11020 @@ def __init__(self):
+                         "questions before the spell ends. The GM answers each question " +
+                         "with one word, such as “yes,” “no,” “maybe,” “never,” " +
+                         "“irrelevant,” or “unclear” (if the entity doesn't know the " +
+                         "answer to the question). If a one-word answer would be " +
+                         "misleading, the GM might instead offer a short phrase as an " +
+                         "answer. ",
+                         at_higher_levels="")
+
+
+class Contagion(spells.Spell):
+    """
+    Contagion Spell
+    SRD p. 130-131
+    Generated
+    """
+
+    def __init__(self):
+        super().__init__(name="Contagion",
+                         spell_lists=[SpellLists.DIVINE],
+                         level=5,
+                         school=SpellSchools.Necromancy,
+                         spell_range="Touch",
+                         verbal_components=True,
+                         somatic_components=True,
+                         material_components_list=None,
+                         duration="7 days",
+                         description="Your touch inflicts disease. Make a melee spell attack against " +
+                         "a creature within your reach. On a hit, you afflict the " +
+                         "creature with a disease of your choice from any of the ones " +
+                         "described below. At the end of each of the target's turns, it " +
+                         "must make a Constitution saving throw. After failing three of " +
+                         "these saving throws, the disease's effects last for the " +
+                         "duration, and the creature stops making these saves. After " +
+                         "succeeding on three of these saving throws, the creature " +
+                         "recovers from the disease, and the spell ends. Since this " +
+                         "spell induces a natural disease in its target, any effect that " +
+                         "removes a disease or otherwise ameliorates a disease's effects " +
+                         "apply to it. Blinding Sickness. Pain grips the creature's " +
+                         "mind, and its eyes turn milky white. The creature has " +
+                         "disadvantage on Wisdom checks and Wisdom saving throws and is " +
+                         "blinded. Filth Fever. A raging fever sweeps through the " +
+                         "creature's body. The creature has disadvantage on Strength " +
+                         "checks, Strength saving throws, and attack rolls that use " +
+                         "Strength. Flesh Rot. The creature's flesh decays. The creature " +
+                         "has disadvantage on Charisma checks and vulnerability to all " +
+                         "damage. Mindfire. The creature's mind becomes feverish. The " +
```

Comment:
A lot of places missing newlines that are explicitly part of the spell description

Reply:
This is a big one, I fixed the rest locally but the parser will need to work on this